tests/queries/0_stateless/01706_optimize_normalize_count_variants.sql

```diff
@@ -0,0 +1,4 @@
+
+set optimize_normalize_count_variants = 1;
+
+explain syntax select count(), count(1), count(-1), sum(1);
```

Comment:
Let's add count(null) to ensure that it is not optimized.

Reply:
Absolutely.